README.md

```diff
@@ -115,54 +115,69 @@ use Rusen::Middleware::RusenRack,
     :outputs => [:io, :email],
     :sections => [:backtrace, :request, :session, :environment],
     :email_prefix => '[ERROR] ',
-    :sender_address => 'some_email@example.com',
-    :exception_recipients => %w(dev_team@example.com test_team@example.com),
+    :sender_address => 'oops@moove-it.com',
+    :exception_recipients => %w(dev_team@moove-it.com),
     :smtp_settings => {
-                        :address              => 'smtp.gmail.com',
-                        :port                 => 587,
-                        :domain               => 'example.org',
-                        :authentication       => :plain,
-                        :user_name            => 'dev_team@moove-it.com',
-                        :password             => 'xxxxxxx',
-                        :enable_starttls_auto => true
-                      }
+      :address              => 'smtp.gmail.com',
+      :port                 => 587,
+      :domain               => 'moove-it.com',
+      :authentication       => :plain,
+      :user_name            => 'dev_team@moove-it.com',
+      :password             => 'xxxxxxx',
+      :enable_starttls_auto => true
+    }
 ```
+
 This will capture any unhandled exception, send an email and write a trace in stdout.
 
 Settings
 ---
+
 ### Outputs
-Currently supported outputs are :io, :lo4r, :pony and :mail. More outputs are easy to add so you can customize Rusen to your needs.
 
-Note: :io will only print to stdout for the time being, but there are plans to extend it to anything that Ruby::IO supports.
+Currently supported outputs are :io, :lo4r, :pony and :mail. More outputs are easy to add so you can customize Rusen to 
+your needs.
+
+Note: :io will only print to stdout for the time being, but there are plans to extend it to anything that Ruby::IO 
+supports.
 
 Pony, lo4r and Mail outputs require additional gems to work.
 
 To use pony add this to your Gemfile:
+
 ```ruby
   gem 'pony'
 ```
 
 To use mail add this to your Gemfile:
+
 ```ruby
   gem 'mail'
 ```
 
 To use log4r add this to your Gemfile:
+
 ```ruby
   gem 'log4r'
 ```
 
 ### Sections
+
 You can choose the output sections simply by setting the appropriate values in the configuration.
 
 ### Exclude if
+
 Here you can pass a block that will receive the error. If the block returns false, then the error will be notified.
 
 ### Email settings
+
 All the email settings are self explanatory, but you can contact me if any of them needs clarification.
```

Comment:
..."but you can contact hello+rusen@moove-it.com if any of them needs clarification"